scripts/codestyle/travis_uncrustify.sh

```diff
@@ -0,0 +1,7 @@
+echo $PWD
+if ! $TRAVIS_PULL_REQUEST; then
+	bash ./run_uncrustify.sh -v $(git rev-parse HEAD~1) $(git rev-parse HEAD)
```

Comment:
we need to filter out codestyle/testcases/ path while codestyle check there fails (so at least this PR will not fail). Let's check `src` only for a while